make-options.json

```diff
@@ -138,6 +138,7 @@
         "TwineAuthenticateV0",
         "UniversalPackagesV0",
         "UseDotNetV2",
+        "UseDotGoV1",
```

Comment:
Should this just be "UseGoV1" instead of "UseDotGoV1"?

Reply:
Yikes, yeah. Copy paste error.